### PR TITLE
[IMP] email_marketing: getting started

### PR DESCRIPTION
**Pull request by samueljlieber**
This PR tests the push process of the BE doc team.

## The following process created this PR:

Updated base branch
```
git checkout 16.0
git pull
git log --oneline
```
Created feature branch
```
git switch -c 16.0-us-docteam-push-sali
```
Stage and commit edits
```
git add .
git commit -m "[IMP] email_marketing: getting started"
```
Push to remote
```
git push --force-with-lease --set-upstream origin 16.0-be-docteam-push-sali
```

### CODE REVIEW

**Comment by samueljlieber** [comment]
Hi @saliodootest this PR is ready for your review

File: content/applications/marketing/email_marketing.rst [hunk 1]
```diff
@@ -5,6 +5,8 @@
 Email marketing
 ===============
 
+This is the first change in the pr for BE push process. Howpeully it works good.
```
Comment (saliodootest) [inline content/applications/marketing/email_marketing.rst]: You have a serious problem.
```suggestion
This is the first change in the pr for BE push process. Hopefully it works well.
```

**Review by saliodootest — CHANGES_REQUESTED** [review]
Ive seen better

**Comment by samueljlieber** [comment]
I can `git pull --rebase` with no issues, updating my local with remote

**Comment by samueljlieber** [comment]
Squashed commits and added a sentence in https://github.com/samueljlieber/dev-documentation/commit/2c8970adbe65e4c08aac61e9217aab272def16a7 using `git push --force-with-lease --set-upstream origin 16.0-be-docteam-push-sali`

**Review by saliodootest — APPROVED** [review]
Looks good to me

File: content/applications/marketing/email_marketing.rst [hunk 2]
```diff
@@ -5,6 +5,9 @@
 Email marketing
 ===============
 
+This is the first change in the pr for BE push process. Hopefully it works well.
+I added this sentence because its cool okay?
```
Comment (saliodootest) [inline content/applications/marketing/email_marketing.rst]: Nope bad sentence i remove
```suggestion
```

**Review by saliodootest — COMMENTED** [review]
Im removing this sentence

**Comment by samueljlieber** [comment]
I am going to make a commit locally ignoring your commit here, and try to push up 😈

**Comment by samueljlieber** [comment]
I recieved an error!!
```
git push --force-with-lease --set-upstream origin 16.0-be-docteam-push-sali

To github.com:samueljlieber/dev-documentation.git
 ! [rejected]            16.0-be-docteam-push-sali -> 16.0-be-docteam-push-sali (stale info)
error: failed to push some refs to 'github.com:samueljlieber/dev-documentation.git'
```

**Comment by samueljlieber** [comment]
This was resolved by `git pull --rebase` fixed merge conflicts (due to same line edits)

**Pull request closed without merging**